docs/source/getting-started/tabular-data/index.rst

```diff
@@ -0,0 +1,103 @@
+:orphan:
+:nosearch:
+
+Working with Tabular Data
+=========================
+
+Motivation 
+----------
+
+In the following examples, we will use real data to demonstrate the utility of OpenDP functions.
+We will use the `Labour Force Survey microdata <https://ec.europa.eu/eurostat/web/microdata/public-microdata/labour-force-survey>`_ released by Eurostat for a few reasons: 
+
+1. **Generality:** The dataset is relatively general, with variables and contexts accessible to users across various domains.
```

Comment:
What does "general" mean though? It's not clear from the explanation. Does it just mean that it's accessible to people everywhere? If so, then is "generality" the right term to describe this property?